code/LeftAndMain.php

```diff
@@ -297,6 +297,9 @@ class LeftAndMain extends Controller implements PermissionProvider
      */
     protected $versionProvider;
 
+    // phpcs:ignore
+    public $_cache_MainMenu;
```

Comment:
```suggestion
    var $_cache_MainMenu;
```
As per https://github.com/silverstripe/silverstripe-framework/pull/10921#pullrequestreview-1592486331